requirements.txt

```diff
@@ -1,2 +1,3 @@
 Jinja2==3.1.1
 ops==2.2.0
+cosl
```

Comment:
Is it okay to not fix the version?

Reply:
Updated, thanks for the thorough review Mert!